smbclient/shutil.py

```diff
@@ -63,6 +63,39 @@
 )
 
 
+def basename(path):
```

Comment:
I don't think we should expose this officially yet, best to just prefix it with `_` if it's used internally here.